WorldCleanup/WorldCleanup.cs

```diff
@@ -60,7 +63,12 @@ public class WorldCleanupMod : MelonMod {
             /* Load our custom UI elements */
             UiExpansion.LoadUiObjects();
 
-            HarmonyInstance.Patch(
+            if (harmony1 == null)
+            {
+                harmony1 = new HarmonyLib.Harmony(BuildInfo.Name + " OnAvatarInstantiateHook");
+            }
```

Comment:
Why do we need an extra harmony instance when Melonloader already provides us with one?